src/main/scala/wdl4s/types/WdlType.scala

```diff
@@ -28,8 +29,12 @@ trait WdlType {
    */
   def coerceRawValue(any: Any): Try[WdlValue] = {
     any match {
-      case v: WdlValue if v.wdlType == this => Success(v)
-      case a if !coercion.isDefinedAt(any) => Failure(new IllegalArgumentException(s"No coercion defined from '$any' of type '${any.getClass}' to ${getClass.getSimpleName}."))
+      case wdlValue: WdlValue if wdlValue.wdlType == this => Success(wdlValue)
+      case wdlValue: WdlValue if !coercion.isDefinedAt(any) => Failure(new IllegalArgumentException(
+        s"No coercion defined from '${ScalaRunTime.stringOf(any, 3)}' of type '${wdlValue.wdlType}' to '$this'."))
```

Comment:
Hehe, good catch also. `scala.runtime.ScalaRuntime` is a lot like `com.sun`, aka "this is undocumented, use this and don't complain if you get burned": https://github.com/scala/scala/blob/v2.11.8/src/library/scala/runtime/ScalaRunTime.scala#L23-L26

But it does exactly what I want, which is to properly render most `Any`s, including arrays-to-string. I didn't like the other hand-crafted suggestions in [this stackoverflow thread](http://stackoverflow.com/questions/3328085/scala-printing-arrays/3328714#3328714).